### DISCUSSION TITLE
How to theme nvimBufferline with Catppuccin

### DISCUSSION THREAD

**Discussion by uncenter** [NotAShelf/nvf#311] [Q&A] [answered]
According to https://github.com/catppuccin/nvim#integrations, I need to configure the `nvimBufferline`/`akinsho/bufferline.nvim` to load after Catppuccin, and set the highlights as follows:

```lua
use "akinsho/bufferline.nvim" {
  after = "catppuccin",
  config = function()
    require("bufferline").setup {
      highlights = require("catppuccin.groups.integrations.bufferline").get()
    }
  end
}
```

I can't figure out a way to do this with the available configuration options.

**Comment by horriblename** [comment, marked as answer]
I don't think it's possible right now, you'll have to wait for #305

**Reply by uncenter** [reply]
Thanks! Don't know how I missed that.

**Reply by uncenter** [reply]
Actually it appears to be working now without any changes on my end besides updating the input? Could https://github.com/NotAShelf/nvf/pull/306 have possibly fixed it?

**Reply by NotAShelf** [reply]
Likely.
